ui/k-info-panel/link_info.kytos

```diff
@@ -0,0 +1,111 @@
+<template>
+    <k-accordion>
+      <k-accordion-item title="Basic Details">
+        <k-property-panel>
+          <k-property-panel-item :name="key" :value="value" 
+          :key="key" v-if="content" v-for="(value, key) in this.details">
+          </k-property-panel-item>
+        </k-property-panel>
+      </k-accordion-item>
+      <k-accordion-item title="Metadata" v-if="Object.keys(this.metadata).length > 0">
+        <div class="metadata_table">
+          <table>
+            <thead>
+              <tr>
+                <th>Key</th>
+                <th>Value</th>  
+              </tr>
+            </thead>
+            <tbody>
+              <tr v-for="(value, key) in this.metadata">
+                <td >{{key}}</td>
+                <td >{{value}}</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </k-accordion-item>
+    </k-accordion>
+</template>
+
+<script>
+ module.exports = {
+   props: ["content"],
+   methods: {
+   },
+   data () {
+     return {
+         metadata: [],
+         details: {'enabled': '',
+                   'active': '',
+                   'id': '',
+                   'endpoint_a': '',
+                   'endpoint_b': '',},
+     }
+   },
+   methods: {
+     get_details: function(){
+       if(this.content === undefined) return
+       var self = this
+       Object.keys(this.details).forEach(function (key) {
+         let value = self.content[key]
+         if (key != 'endpoint_a' && key != 'endpoint_b') {
+             self.details[key] = String(value)
+         }
+         else {
+             self.details[key] = String(value['id'])
+         }
+       });
+       this.get_metadata()
+     },
+     get_metadata: function(){
+       if(this.content === undefined) return
+       this.metadata = this.content.metadata
+     },
+   },
+   mounted () {
+       this.get_details()
+   },
+   watch:{
+       content(){
+           if (this.content){
+               this.get_details()
+           }
+       }
+   }
+ }
+</script>
+
+<style>
+.metadata_table {
+  color: #ccc;
+  max-height: 250px;
```

Comment:
@Alopalao could you remove the `max-height`? since this table will always be the last component, I think it's desirable to let it expand. Also, could you add `overflow-x: auto;`? In case there are long values it should provide a horizontal scroll.

Reply:
Fixed on commit [38f50595d8b02003539a3f0769dcdc68c8ee94ad](https://github.com/kytos-ng/topology/pull/73/commits/38f50595d8b02003539a3f0769dcdc68c8ee94ad)